features/backend/deleting_account.feature

```diff
@@ -0,0 +1,26 @@
+@ui-user
+Feature: Deleting the account in backend
+    In order to delete Customer account on demand
+    As an Administrator
+    I want to be able to remove Customer account details from the system
+
+    Background:
+        Given there is user "theodore@example.com" identified by "pswd"
+        And there is an administrator account
+        And I am logged in as administrator
```

Comment:
This step should be merged with the one above (create default admin account and login).